### ISSUE TITLE
TaskGroup.isRunning never returns a boolean

### ISSUE THREAD

**Issue by antidis** [machty/ember-concurrency#113]
`isRunning` on a TaskGroup returns an `Ember.computed.or` of two integers. Unfortunately, `Ember.computed.or` performs `||` which does not guarantee a `boolean`, so this leads to this truth table for `numRunning` and `numQueued`:

```
| numRunning | numQueued | isRunning |
|------------|-----------|-----------|
| 0          | 0         | 0         |
| 1          | 0         | 1         |
| 0          | 1         | 1         |
| 1          | 1         | 1         |
| 2          | 1         | 2         |
| 1          | 2         | 1         |
| ...        | ...       | ...       |
```

PR to fix: https://github.com/machty/ember-concurrency/pull/112

**Comment by machty** [comment]
Fixed in 0.7.18. Thanks for the code!

**Comment by antidis** [comment]
👍
